src/library/Server/Manager/Directadmin.php

```diff
@@ -315,152 +394,176 @@ public function modifyAccount(Server_Account $a)
 
+    /**
+     * This method is used to verify the authentication credentials for the DirectAdmin server.
+     * It sends a request to the server with the 'API_VERIFY_PASSWORD' command and the username and password from the configuration.
+     * The response from the server is stored in the $response variable, but is not used in this method.
+     * The method always returns true, regardless of the server's response.
+     *
+     * @return bool Always returns true.
+     * @throws Server_Exception If there is an error while sending the request to the server.
+     */
     private function checkAuth()
     {
-        $r = $this->_request('CMD_API_VERIFY_PASSWORD', array(
+        $response = $this->_request('API_VERIFY_PASSWORD', array(
             'user' => $this->_config['username'],
             'passwd' => $this->_config['password']
         ));
 
         return true;
     }
 
-    private function getPackages()
-    {
-        return $this->_request('CMD_API_PACKAGES_USER');
-    }
-
+    /**
+     * @throws Server_Exception
+     */
     private function getIps()
     {
-        $results = $this->_request('CMD_API_SHOW_RESELLER_IPS');
+        $results = $this->_request('API_SHOW_RESELLER_IPS');
         return $results['list'] ?? array();
     }
 
     /**
      * @param string $command
+     * @param array $fields
+     * @param bool $post
+     * @return array
+     * @throws Server_Exception
      */
-    private function _request($command, $fields = array(), $post = true)
+    private function _request(string $command, array $fields = array(), bool $post = true): array
     {
         $host = $this->_config['host'];
         $protocol = $this->_config['secure'] ? 'https://' : 'http://';
 
-        $fieldstring = http_build_query($fields);
+        $field_string = http_build_query($fields);
 
-        $httpClient = $this->getHttpClient()->withOptions([
-            'verify_peer'   => false,
-            'verify_host'   => false,
-            'timeout'       => 60,
+        $http_client = $this->getHttpClient()->withOptions([
             'auth_basic'    => [ $this->_config['username'], $this->_config['password'] ],
+            'timeout'       => 60,
+            'verify_host'   => false,
+            'verify_peer'   => false,
         ]);
 
-        $url = $protocol . $host . ':'. $this -> _getPort().'/' . $command . '?' . $fieldstring;
+        $url = $protocol . $host . ':'. $this -> _getPort().'/CMD_' . $command . '?' . $field_string;
         $this->getLog()->debug($url);
 
         try {
-            if($post) {
-                $request = $httpClient->request('POST', $url, [
-                    'body'  => $fields,
-                ]);
-            } else {
-                $request = $httpClient->request('GET', $url);
-            }
-
+            // If it's a POST request, include the fields in the request body
+            $request = $http_client->request($post ? 'POST' : 'GET', $url, $post ? ['body' => $fields] : []);
```

Comment:
I personally think this change makes the code far less readable and would prefer it if the original code was kept.

Reply:
Sure thing, no problem at all